docs/sources/guides/whats-new-in-v6-5.md

```diff
@@ -35,6 +38,65 @@ Grafana 6.5 comes with a lot of new features and enhancements.
 
 More details of above and highlights will be added as we're getting closer to the stable release.
 
+### CloudWatch data source improvements
+
+In this release, several feature improvements and additions were made in the CloudWatch data source. This work has been done in collaboration with the Amazon CloudWatch team.
+
+#### GetMetricData API
+
+For Grafana version 6.5 or higher, all API requests to GetMetricStatistics have been replaced with calls to GetMetricData, following Amazon’s [best practice to use the GetMetricData API](https://aws.amazon.com/premiumsupport/knowledge-center/cloudwatch-getmetricdata-api) instead of GetMetricStatistics, because data can be retrieved faster at scale with GetMetricData. This change provides better support for CloudWatch metric math and enables the use of automatic search expressions.
+
+While GetMetricStatistics qualified for the CloudWatch API free tier, this is not the case for GetMetricData calls. For more information, please refer to the [CloudWatch pricing page](https://aws.amazon.com/cloudwatch/pricing/).
+
+#### Dynamic queries using dimension wildcards
+
+In Grafana 6.5 or higher, you’re able to monitor a dynamic list of metrics by using the asterisk (\*) wildcard for any one or multiple dimension values.
+
+In the example below, all metrics in the namespace `AWS/EC2` with a metric name of `CPUUtilization` and ANY value for the `InstanceId` dimension are queried. This can help you monitor metrics for AWS resources, like EC2 instances or containers. For example, when new instances get created as part of an auto scaling event, they will automatically appear in the graph without you having to track the new instance IDs. This capability is currently limited to retrieving up to 100 metrics. You can click on `Show Query Preview` to see the search expression that is automatically built to support wildcards. To learn more about search expressions, visit the [CloudWatch documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/search-expression-syntax.html).
```

Comment:
Do you need to mention the 100 metric limit here? Feels like something that does not impact most users and can be in the docs.

Reply:
Link to upgrading.md, `/installation/upgrading/#upgrading-to-v6-5` instead.

Reply:
> Do you need to mention the 100 metric limit here? Feels like something that does not impact most users and can be in the docs.

I agree! 



> Link to upgrading.md, `/installation/upgrading/#upgrading-to-v6-5` instead.

Where?

Reply:
Not sure where. Thought in relation to 100 metric limit. But should be linked somewhere in regards to the breaking changes introduced.